src/pages/cart/Cart.scss

```
@@ -1,3 +1,261 @@
-div{
-    font-size: 15px;
-}
\ No newline at end of file
+@mixin flex($justify, $align) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.cartMain {
+  @include flex(center, null);
+  height: 100vh;
+
+  .cartContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+  }
+
+  .cartTitle {
+    @include flex(center, center);
+    font-size: 35px;
+    width: 860px;
+  }
+
+  .cartList {
+    float: left;
+    width: 860px;
+    margin-top: 50px;
+    border-top: 1px solid black;
+    .cartListTitle {
+      @include flex(space-between, center);
+      width: 100%;
+      height: 55px;
+      border-bottom: 1px solid #e1dedf;
+      .cartListTitleBlankImg {
+        width: 109px;
+      }
```

```suggestion
    .cartListTitle {
      @include flex(space-between, center);
      width: 100%;
      height: 55px;
      border-bottom: 1px solid #e1dedf;
      &BlankImg {
        width: 109px;
      }
```

이름이 지속적으로 중복될 때에는 부모 선택자인 `&`를 적극 활용해보시길 바랍니다.

---

많은 className & 처리 하였습니다.